modules/calendar/calendar.php

```diff
@@ -749,111 +761,116 @@ function view_calendar() {
 
 				<?php
 				$current_month = date_i18n( 'F', strtotime( $filters['start_date'] ) );
-				for( $current_week = 1; $current_week <= $this->total_weeks; $current_week++ ):
+				for ( $current_week = 1; $current_week <= $this->total_weeks; $current_week++ ) :
 					// We need to set the object variable for our posts_where filter
 					$this->current_week = $current_week;
 					$week_posts = $this->get_calendar_posts_for_week( $post_query_args );
 					$date_format = 'Y-m-d';
 					$week_single_date = $this->get_beginning_of_week( $filters['start_date'], $date_format, $current_week );
 					$week_dates = array();
 					$split_month = false;
-					for ( $i = 0 ; $i < 7; $i++ ) {
-						$week_dates[$i] = $week_single_date;
+					for ( $i = 0; $i < 7; $i++ ) {
+						$week_dates[ $i ] = $week_single_date;
 						$single_date_month = date_i18n( 'F', strtotime( $week_single_date ) );
 						if ( $single_date_month != $current_month ) {
 							$split_month = $single_date_month;
 							$current_month = $single_date_month;
 						}
-						$week_single_date = date( 'Y-m-d', strtotime( "+1 day", strtotime( $week_single_date ) ) );
+						$week_single_date = gmdate( 'Y-m-d', strtotime( '+1 day', strtotime( $week_single_date ) ) );
```

Comment:
@ingeniumed Changing from `date` to `gmdate` here leads to a date that's not in the timezone of the current site. For here (and other places?) where `date` changed to `gmdate`, did we ensure behavior didn't change?

Reply:
Reverted this as discussed